syntax = "proto3";

message O1MeasurementsReport {
  string imsi = 1;
  int32 rx_power_avg = 2;
  int32 rx_power_tot = 3;
  int32 n0_power_avg = 4;
  int32 n0_power_tot = 5;
  int32 rx_rssi_dBm = 6;
  int32 ssb_rsrp_dBm = 7;
  int32 mcs = 8;
}

